tappio/scripts/missing_accounts.py

```diff
@@ -33,22 +38,22 @@ def missing_accounts(*input_filenames):
     flat_accounts = dict((filename, flatten_accounts(document.accounts)) for (filename, document) in documents)
 
     all_accounts = dict()
-    map(all_accounts.update, flat_accounts.itervalues())
+    list(map(all_accounts.update, six.itervalues(flat_accounts)))
```

Comment:
`map` was used here for side effects. We should convert this to an explicit `for` loop.

Reply:
Revisiting this after a while (and, I confess, after a glass or two of wine ...) and bearing in mind that explicit is better than implicit, your ugestion of a for loop seems more sensible.